cluster-provision/k8s/1.22/provision.sh

```diff
@@ -103,7 +103,9 @@ baseurl=https://storage.googleapis.com/kubevirtci-crio-mirror/devel_kubic_libcon
 gpgcheck=0
 enabled=1
 EOF
-dnf install -y cri-o
+# TODO: Remove the package pinning once cri-o support the new 'keyPaths' key
+dnf install -y cri-o containers-common-1-23.module_el8.7.0+1106+45480ee0.x86_64
+echo "" >> /etc/containers/policy.json
```

Comment:
Is this still required?

Reply:
Yes - the later install of podman upgrades the containers-common package - it prevents policy.json for being updated as part of that upgrade.